src/sdk/config_fs.js

```diff
@@ -472,6 +472,20 @@ class ConfigFS {
         return path.join(this.old_accounts_dir_path, this.json(account_name));
     }
 
+    /**
+     * Returns the path to a connection file
+     * @param {string} connection_name name of the desired connection
+     * @returns {string}
+     */
+    get_connection_path_by_name(connection_name) {
+        return path.join(this.connections_dir_path, this.json(connection_name));
+    }
+
+    async get_connection_by_name(connection_name) {
+        const filepath = path.join(this.connections_dir_path, this.json(connection_name));
+        return await this.get_config_data(filepath);
+    }
+
```

Comment:
You added it under:

    ///////////////////////////////////////
    ////// ACCOUNT CONFIG DIR FUNCS  //////
    ///////////////////////////////////////

Please create a title for the connection and make sure to add it in appropriate place (I think the method before and after are related to accounts).